secure/storage/github/src/lib.rs

```diff
@@ -173,7 +174,10 @@ impl Client {
             .set("Authorization", &format!("token {}", self.token))
             .set(ACCEPT_HEADER, ACCEPT_VALUE)
             .timeout_connect(TIMEOUT);
-        request
+        match env::var("https_proxy") {
+            Ok(proxy) => request.set_proxy(ureq::Proxy::new(proxy).unwrap()).build(),
```

Comment:
maybe use an expect here => expect("Unable to parse https_proxy environmental variable")